layouts/partials/footer.html

```diff
@@ -0,0 +1,137 @@
+        <img class="logo-eclipse-default img-responsive" width="140" typeof="foaf:Image" src="https://www.openmdm.org/sites/all/themes/solstice/_themes/solstice_openmdm/logo.png" alt="" title=""><p class="padding-top-15">OpenMDM is a Working Group of The Eclipse Foundation.</p><hr><a href="https://www.eclipse.org"><img class="logo-eclipse-default img-responsive" width="160" typeof="foaf:Image" src="https://www.openmdm.org/sites/all/themes/solstice/public/images/logo/eclipse-foundation-grey-orange.svg" alt="" title="Eclipse Foundation"></a>
+      </section>
+      <div class="col-sm-24 margin-top-20">
+        <div class="row">
+          <div id="copyright" class="col-md-16">
+            <p id="copyright-text">Copyright © Eclipse Foundation, Inc. All Rights Reserved.</p>
+          </div>
+          <div class="col-md-8 social-media">
+            <ul class="list-inline">
+              <li>
+                <a class="social-media-link fa-stack fa-lg" href="https://twitter.com/EclipseFdn">
+                  <i class="fa fa-circle-thin fa-stack-2x"></i>
+                  <i class="fa fa-twitter fa-stack-1x"></i>
+                </a>
+              </li>
+              <li>
+                <a class="social-media-link fa-stack fa-lg" href="https://www.facebook.com/eclipse.org">
+                  <i class="fa fa-circle-thin fa-stack-2x"></i>
+                  <i class="fa fa-facebook fa-stack-1x"></i>
+                </a>
+              </li>
+              <li>
+                <a class="social-media-link fa-stack fa-lg" href="https://www.youtube.com/user/EclipseFdn">
+                  <i class="fa fa-circle-thin fa-stack-2x"></i>
+                  <i class="fa fa-youtube fa-stack-1x"></i>
+                </a>
+              </li>
+              <li>
+                <a class="social-media-link fa-stack fa-lg" href="https://www.linkedin.com/company/eclipse-foundation">
+                  <i class="fa fa-circle-thin fa-stack-2x"></i>
+                  <i class="fa fa-linkedin fa-stack-1x"></i>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div><a href="#" class="scrollup" style="display: inline;">Back to the top</a>
+    </div>
+  </div>
+</footer>
+
+<!-- Placed at the end of the document so the pages load faster -->
+{{- with .Site.Params.js | default "https://www.eclipse.org/eclipse.org-common/themes/solstice/public/javascript/main.min.js"}}
+<script src="{{ . | absURL }}"></script>
+{{ end }}
+
+<script>
+  (function($, document) {
+  // Eclipse News and events
+  {{ range (.Page.Scratch.Get "news-container-ids") }}
+    $("#{{ . }}").eclipseFdnApi({
+      type: "newsItems"
+    });
+  {{ end }}
+
+  {{ if ne .Page.Params.show_featured_footer false }}
+    $("{{ .Site.Params.featured_footer_selector | default "#featured-footer" }}").eclipseFdnApi({
+      type: "featuredFooter"
+    });
+  {{ end }}
+  })(jQuery, document);
+  
+  (function($, document) {
```

Comment:
We likely don't need this. See jakarta.ee impl, specifically the site config params:

```
  call_for_action_text = "Download"
  call_for_action_url = "/release/"
  call_for_action_icon = "fa-download"
```

Reply:
But using that, can I put the download button above the nav? like this:
![image](https://user-images.githubusercontent.com/39588094/96009316-85ad7300-0e0e-11eb-971d-2c0923034d29.png)

Reply:
It floats like that because they put too many items in their navigation. If you copy everything the way its done on that side, it'll likely float the same, but that isn't an intentional design decision.

Reply:
Have checked, it won't float up even I add more navs.  M not sure which one should we use? Keep it at the top or just add at the end? 

![image](https://user-images.githubusercontent.com/39588094/96024165-610ec680-0e21-11eb-95a6-f3df5a9d27e2.png)

Reply:
Example using Jakarta.ee (which is using the mechanism I mentioned)
![image](https://user-images.githubusercontent.com/23018241/96033727-7807e580-0e2e-11eb-95cc-9bf6089ec94b.png)


Considering that it isn't intentional and is a side effect? We should make use of the CTA config values to keep it consistent. When upgrading this site, we shouldn't be aiming for exact duplicate, as there are things that will have changed and updated since this site was first created, and small shifts from Drupal to Hugo.

Reply:
How did you make it work on jakarta EE? Adding a nav_more?

Reply:
As discussed, we can fix this by adding a clear:both on #main-menu.

Reply:
I think the other reason is that their logo is longer so that we probably still try to put the button at the top, but instead of using jquery, Chris just showed me a better way to add a css to fix it.